x-pack/plugins/cases/public/types.ts

```diff
@@ -52,6 +53,12 @@ export type StartServices = CoreStart &
   };
 
 export interface CasesUiStart {
+  /**
+   * Determines whether the current user has the ability to use Cases.
```

Comment:
Maybe you can extend a bit the explanation to add what I described in https://github.com/elastic/kibana/pull/120321/files#r773919983